### PR TITLE
Update glossary.mdx

### PR DESCRIPTION
**Pull request by tsoysergey**
Corrected spelling errors

### CODE REVIEW

File: website/content/docs/install/glossary.mdx [hunk 1]
```diff
@@ -151,7 +151,7 @@ broader cloud-native community of cloud engineers.  Read more
 
 ## Custom Resource Definition (CRD) 	
 Custom resources are the extensions of the Kubernetes API.  A Custom Resource 
-Definition (CRD) file allows a user to define their own custom resources and 
+Definition (CRD) file allows users to define their own custom resources and
```
Comment (dnephin) [inline website/content/docs/install/glossary.mdx:154]:
```suggestion
Definition (CRD) file allows a user to define their own custom resources and 
```

**Review by karl-cardenas-coding — APPROVED** [review]
lgtm

**Comment by dnephin** [comment]
Thank you for the PR! After some discussion we decided to only keep the second edit, as the first one seemed to read a bit better as it was. 

There are some CI failures, but I believe those are due to this being an older branch. I don't think the changes in this PR can cause any failures, so I'll merge.

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/539758.

**Comment by hc-github-team-consul-core** [comment]
:cherries::x: Cherry pick of commit 0a83964a5a04bca69cbf37fa0f6ee028f82b9951 onto `stable-website` failed! [Build Log](https://circleci.com/gh/hashicorp/consul/539758)

**Comment by hc-github-team-consul-core** [comment]
:cherries::x: Cherry pick of commit 0a83964a5a04bca69cbf37fa0f6ee028f82b9951 onto `release/1.11.x` failed! [Build Log](https://circleci.com/gh/hashicorp/consul/539758)

**Comment by hc-github-team-consul-core** [comment]
:cherries::x: Cherry pick of commit 0a83964a5a04bca69cbf37fa0f6ee028f82b9951 onto `release/1.10.x` failed! [Build Log](https://circleci.com/gh/hashicorp/consul/539758)

**Comment by hc-github-team-consul-core** [comment]
:cherries::x: Cherry pick of commit 0a83964a5a04bca69cbf37fa0f6ee028f82b9951 onto `release/1.9.x` failed! [Build Log](https://circleci.com/gh/hashicorp/consul/539758)

**Comment by hc-github-team-consul-core** [comment]
:cherries::x: Cherry pick of commit 0a83964a5a04bca69cbf37fa0f6ee028f82b9951 onto `release/1.8.x` failed! [Build Log](https://circleci.com/gh/hashicorp/consul/539758)

**Comment by dnephin** [comment]
Backport seemed to have failed because there is no diff, so I think we're good. Nothing to manually backport.

**Pull request merged**